src/App.js

```diff
@@ -1,6 +1,7 @@
 import React, { Component } from 'react';
 import { HomePage } from './HomePage/HomePage.js';
 import { getNewDog } from './apiCalls/dogcalls.js';
+import { Route, Switch } from 'react-router-dom';
```

Comment:
I like using switch, but do I need it? Can I get away with something simpler? Is this the simplest?